services/core-api/app/api/incidents/response_models.py

```diff
@@ -4,7 +4,9 @@
 MINE_INCIDENT_CATEGORY_MODEL = api.model(
     'Mine Incident Category', {
         'mine_incident_category_code': fields.String,
-        'description': fields.String
+        'description': fields.String,
+        'display_order': fields.Integer,
+        'active_ind': fields.Boolean
```

Comment:
Do we use active_ind on other response models?

Reply:
I'm not sure but Jason requested I put it in for future use.